applications/nrf_desktop/configuration/nrf54lm20pdk_nrf54lm20a_cpuapp/prj.conf

```diff
@@ -0,0 +1,141 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_ROLE_HID_PERIPHERAL=y
+CONFIG_DESKTOP_PERIPHERAL_TYPE_MOUSE=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DE
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_MOUSE=y
+
+# Application supports 2 HID subscribers: HID over GATT (BLE) and a single USB HID instance
+CONFIG_DESKTOP_HID_STATE_SUBSCRIBER_COUNT=2
+
+CONFIG_DESKTOP_MOTION_BUTTONS_ENABLE=y
+CONFIG_DESKTOP_MOTION_BUTTONS_UP_KEY_ID=0
+CONFIG_DESKTOP_MOTION_BUTTONS_DOWN_KEY_ID=3
+CONFIG_DESKTOP_MOTION_BUTTONS_LEFT_KEY_ID=2
+CONFIG_DESKTOP_MOTION_BUTTONS_RIGHT_KEY_ID=1
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_DESKTOP_USB_ENABLE=y
+# Enable the only USB stack that supports USBHS
+CONFIG_DESKTOP_USB_STACK_NEXT=y
+
+CONFIG_DESKTOP_BLE_USE_DEFAULT_ID=y
+
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0000
+CONFIG_DESKTOP_BLE_PEER_ERASE_ON_START=y
+
+CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
+CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT=y
+
+# START of workaround for issue NCSDK-33313
+# Disable CRACEN for all but RNG and switch to software crypto.
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+CONFIG_PSA_CRYPTO_DRIVER_CRACEN=y
+
+CONFIG_PSA_USE_CRACEN_CIPHER_DRIVER=n
+CONFIG_PSA_USE_CRACEN_AEAD_DRIVER=n
+CONFIG_PSA_USE_CRACEN_HASH_DRIVER=n
+CONFIG_PSA_USE_CRACEN_MAC_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_AGREEMENT_DRIVER=n
+CONFIG_PSA_USE_CRACEN_ASYMMETRIC_ENCRYPTION_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_MANAGEMENT_DRIVER=n
+CONFIG_PSA_USE_CRACEN_KEY_DERIVATION_DRIVER=n
+CONFIG_PSA_USE_CRACEN_PAKE_DRIVER=n
+CONFIG_PSA_USE_CRACEN_ASYMMETRIC_SIGNATURE_DRIVER=n
+# END of workaround for issue NCSDK-33313
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=3584
+CONFIG_ISR_STACK_SIZE=2560
+CONFIG_MAIN_STACK_SIZE=2816
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+CONFIG_IDLE_STACK_SIZE=512
+
+CONFIG_BOOT_BANNER=n
+CONFIG_NCS_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=2560
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+CONFIG_GPIO=y
+
+# Optimize for size to reduce memory footprint.
+CONFIG_SIZE_OPTIMIZATIONS=y
```

Comment:
I think it's gaming grade mouse use-case. I would suggest to optimize for speed (also for release configuration)

Reply:
discussed offline